docs/docs/schema.md

```diff
@@ -0,0 +1,367 @@
+---
+
+title: Schema
+description: Define the schema of your database.
+
+---
+
+## Overview
+
+In Drift, your schema is represented as a table. Each column in the table represents a field in your data class.
+
+Take this example of a table that stores superheroes:
+
+| ID  | Name      | Secret Name  | Age    | Height |
+| --- | --------- | ------------ | ------ | ------ |
+| 1   | Superman  | Clark Kent   | 35     | 6'2"   |
+| 2   | Batman    | Bruce Wayne  | 40     | 6'0"   |
+| 3   | Spiderman | Peter Parker | 25     | 5'10"  |
+| 4   | Ironman   | Tony Stark   | `null` | 6'1"   |
+| 5   | Thor      | `null`       | 1500   | `null` |
+
+
+Here we have a table with 5 columns: `ID`, `Name`, `Secret Name`, `Age`, and `Height`. Each row in the table represents a single superhero.
+
+Also, we need to limit how the data is stored, for instance:
+
+- `ID` and `Name` should be unique for each row.
+- `Secret Name`, `Age`, and `Height` are optional fields.
+
+You can easily define a table in Drift using the following syntax:
+
+{{ load_snippet('superhero_schema','lib/snippets/schema.dart.excerpt.json') }}
+
+After defining the schema, add the table to the database. 
+
+{{ load_snippet('superhero_database','lib/snippets/schema.dart.excerpt.json') }}
+
+
+Now, run the code generator:
+```bash
+dart run build_runner build
+```
+
+<h4>Congratulations! 🎉🎉</h4>
+
+You've successfully defined the schema for your database.
+You can now use the generated code to interact with your database.
+
+{{ load_snippet('superhero_query','lib/snippets/schema.dart.excerpt.json') }}
```

Comment:
I wonder if we should have tabs here to show how to do this with manager and with the regular query builder (happy to write half of the snippets in that case :D)

Reply:
I plan on breaking up the query docs into 3 parts

Manager API
Core API
SQL APi

The Manager API is the simplest, so I want that to be the 1st thing users see.
In the query docs that I'm in middle of writing I reference the Core API docs for more complex operations.